### PR TITLE
KULRICE-14160 Created GitHub specific README.md

### PR DESCRIPTION
**Pull request by cniesen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Kuali Rice
+==========
+Kuali Rice is an application development framework and middle ware suite developed for and by the higher education community.  Kuali Rice is being used by several Kuali universiy enterprise applications, for its build in eDocLite document routing, and as basis for custom applications.
```
Comment (ewestfal) [inline README.md]: "middle ware" should be one word

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+More info about Kuali Rice at http://www.kuali.org/rice.
+
+Where to find the official version?
+-----------------------------------
+Endusers can download Kuali Rice at http://www.kuali.org/download/rice
```
Comment (ewestfal) [inline README.md]: "End users" should be two words.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Developers can contribute to Kuali Rice at https://github.com/kuali/rice
+
+Additional Resources
+--------------------
+* Wiki:https://wiki.kuali.org/display/KULRICE/Home
+* Issue tracking: https://github.com/kuali/rice/issues
```
Comment (ewestfal) [inline README.md]: We need to point them at our Jira for issue tracking

File: README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+-------------
+Contributions are welcome. The Rice Project strives for a simple and quick contribution process.  However, there are some specific requirements we need to enforce:
+* Please report all security issues to security@kuali.org. 
+* It is recommended to contact the Kuali Rice development team before starting work on a pull request.  Send the contribution proposal to the collaboration mailing list (rice.collab@kuali.org). The Kuali Rice team will evaluate the proposal, assign a Jira issue, and respond via email.
+* Every contributor must have a Contributor License Agreement (CLA on File with Kuali).  To submit the CLA:
+  * Create an account at http://kis.kuali.org/ (details: [Create a KIS account] (https://wiki.kuali.org/display/KULFOUND/Create+a+KIS+account)).
+  * Sign the [Contributor License Agreement] (https://docs.google.com/a/kuali.org/file/d/0B98dhdqKa_2hNzk5MzhhODgtYzM2Ni00MmUyLWFlYzYtNmJhMDdkZGM2MTI4/edit), scan and send it to help@kuali.org (details: [License Agreement Process (CLA / CCLA)] (https://wiki.kuali.org/pages/viewpage.action?pageId=313879878)).
+* Create a separate pull request for each feature.
+* Squash pull request into one commit.
```
Comment (ewestfal) [inline README.md]: Can we provide a link or something with information on how to "squash" a pull request into one commit?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* Prefix each commit message with the Jira issue identifier.
```
Comment (ewestfal) [inline README.md]: I would provide an example, so "(i.e. KULRICE-1234)" or something along those lines

File: README.md (hunk 4 above)
Comment (cniesen) [inline README.md]: A direct link to https://wiki.kuali.org/pages/viewpage.action?pageId=338976534#Git/GitHubWorkflow,PoliciesandUsage-1.4.5SquashingChangeLog or just to the general GitHubworkflow, Policies and Usage page (top)?

**Comment by cniesen** [comment]
Thanks @ewestfal, I made the suggested changes.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+Kuali Rice
+==========
+Kuali Rice is an application development framework and middleware suite developed for and by the higher education community.  Kuali Rice is being used by several Kuali universiy enterprise applications, for its build in eDocLite document routing, and as basis for custom applications.
```
Comment (shahess) [inline README.md]: "for its build in eDocLite document routing"  is confusing to me.  Do you mean  "to build eDocLite documents"  or "for building eDocLite documents"  ?

Comment (cniesen) [inline README.md]: No, I meant build-in.  Would a dash clarify it or still cause some ambiguity?

Comment (shahess) [inline README.md]: Oh, I see what you mean, the eDocLite routing functionality is built in.  I was just reading it wrong, I don't think the dash is needed.

**Pull request merged**